tests/nativeapp/test_factory_v11.py

```diff
@@ -0,0 +1,73 @@
+from pathlib import Path
+
+from tests.nativeapp.factories import PdfV11Factory, ProjectV11Factory
+
+
```

Comment:
can we add a test to this file to test a factory with template tags?

for example
```python
def test_templates(temp_dir):
    pdf_res = PdfV11Factory(
        native_app__name="myapp_<% ctx.env.FOO %>",
        native_app__env__FOO="bar",
    )
    assert pdf_res.yml["native_app"]["name"] == "myapp_bar"
```
or something like that

Reply:
added test for templating